### PR TITLE
adding FWF export,

### PR DESCRIPTION
**Pull request by ZenoLC**
* refactoring science europe template

refs #77606, #77607, #79403

### CODE REVIEW

**Comment by ZenoLC** [comment]
currently hardcoded to export FWF document, 
adaptive system still pending.

File: src/main/java/at/ac/tuwien/damap/conversion/ExportFWFTemplate.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.*;
+
+@ApplicationScoped
+@JBossLog
+public class ExportFWFTemplate extends ScienceEuropeComponents{
+
+    @Inject
+    TemplateFileBrokerService templateFileBrokerService;
```
Comment (clarascu) [inline src/main/java/at/ac/tuwien/damap/conversion/ExportFWFTemplate.java:15]: Same variable as in `AbstractExportTemplate` (see sonar analysis).

File: src/main/java/at/ac/tuwien/damap/conversion/AbstractExportTemplate.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    @Inject
+    TemplateFileBrokerService templateFileBrokerService;
+
+    @Inject
+    LoadResourceService loadResourceService;
+
+    protected Map<String, String> replacements = new HashMap<>();
+    protected Map<String, String> footerMap = new HashMap<>();
+    //Convert the date for readable format for the document
+    protected final SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd");
```
Comment (clarascu) [inline src/main/java/at/ac/tuwien/damap/conversion/AbstractExportTemplate.java:36]: Same variable as in `DocumentConversionService` (see sonar analysis).

**Review by clarascu — CHANGES_REQUESTED** [review]
It might be helpful to rename `DocumentConversionService`, `AbstractExportTemplate` & `ScienceEuropeComponents` to better capture their purpose.

**Comment by ZenoLC** [comment]
classes were renamed to better reflect their functionality.

**Review by clarascu — APPROVED** [review]

**Pull request merged**